allennlp/service/server_sanic.py

```diff
@@ -8,34 +8,36 @@ def run(port: int) -> None:
     print("Starting a sanic server on port {}.".format(port))
     app = make_app()
     # TODO(joelgrus): make this configurable
-    app.servables = ServableCollection.default()
+    app.predictors = PredictorCollection.default()
```

Comment:
Would it make sense to just put the `from_params()` logic inside of the app itself, and remove the need for `PredictorCollection` entirely?